plugins/buidler.plugin.js

```diff
@@ -91,6 +125,9 @@ function plugin() {
         } = utils.assembleFiles(config, skipFiles);
```

Comment:
The plugin obtains the contracts that must be instrumented here. If we instrument all source code that we see in `TASK_COMPILE_SOLIDITY_GET_COMPILER_INPUT` instead, it might be enough to instrument `node_module` contract dependencies too. What do you think, @alcuadrado @fvictorio?